src/java.base/share/classes/sun/security/timestamp/HttpTimestamper.java

```diff
@@ -112,27 +111,19 @@ public TSResponse generateTimestamp(TSRequest tsQuery) throws IOException {
         connection.connect(); // No HTTP authentication is performed
 
         // Send the request
-        DataOutputStream output = null;
-        try {
-            output = new DataOutputStream(connection.getOutputStream());
+        try (var output = new DataOutputStream(connection.getOutputStream())) {
```

Comment:
Just wondering why you chose to switch stylistically to using the var keyword.

Reply:
That's one of the places where I personally would always use the var keyword too: it makes the line shorter and the type is already clearly visible in the RHS of the assignment, so repeating it on the left hand side does not bring much value...

Reply:
Just interesting that the style wasn't consistent.  Both ways are fine by me.

Reply:
I used `var` only in places, where it allows to reduce line length to be less than 80.